<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgQuickLoad</class>
 <widget class="QDialog" name="DlgQuickLoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btnShowJnts">
     <property name="text">
      <string>Show Joints</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnShowIkCtrls">
       <property name="text">
        <string>Show Ik Ctrls</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnShowFkCtrls">
       <property name="text">
        <string>Show Fk Ctrls</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnShowOtherCtrls">
       <property name="text">
        <string>Show Other Ctrls</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>53</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
